scripts/lua/modules/alert_store/flow_alert_store.lua

```diff
@@ -904,6 +894,47 @@ function flow_alert_store:format_record(value, no_html)
         end
     end
 
+    -- add additional flow related info
+    if (verbose == "true") then
```

Comment:
please use "if verbose then" (do not use a string, convert the string to a boolean in the caller)